### PR TITLE
Add DB schema

### PR DESCRIPTION
**Pull request by Chabare**
Let's go into the final review round.
I think we discussed most of the things in #1 already.

From my POV, it's ready.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        - checkout
+        - run:
+            name: Replace password in flyway.conf
+            command: sed -i -e "s/{{password}}/${OPENCOCKTAIL_CIRCLECI_PASSWORD}/g" flyway.conf
+        - run:
+            name: Replace password in flyway.conf
+            command: sed -i -e "s/{{base_url}}/${OPENCOCKTAIL_CIRCLECI_BASE_URL}/g" flyway.conf
+        - run:
+            name: Copy sql to flyway folder
+            command: mv sql/* /flyway/sql/
```
Comment (BjoernPetersen) [inline .circleci/config.yml]: This is unnecessary.
Just add `flyway.locations=filesystem:sql` to the flyway.conf.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+flyway.conf
```
Comment (BjoernPetersen) [inline .gitignore]: `flyway.conf` is checked into git, and should be. Although maybe it should be called something like `circleCiFlyway.conf` or, better yet, reside in the `.circleci/` directory.

File: sql/V2.4__Insert_drinks.sql [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	'Hendricks Gin is best paired with a cucumber and a mold tonic water.',
+	(SELECT id FROM drink_categories WHERE drink_categories.name = 'Highballs'),
+	(SELECT id FROM glasses WHERE glasses.name = 'Highball'),
+	true
+);
+
+INSERT INTO drinks (name, image_link, description, revision_date, notes, category_id, glass_id, ice_cubes)
+VALUES (
+	'Vodka-O',
+	'TODO',
```
Comment (BjoernPetersen) [inline sql/V2.4__Insert_drinks.sql]: The TODOs here should be replaced by distinguishable values.

File: sql/V2.4__Insert_drinks.sql [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+	'TODO',
+	1535587200,
+	'TODO',
+	(SELECT id FROM drink_categories WHERE drink_categories.name = 'Highballs'), -- Is this correct?
+	(SELECT id FROM glasses WHERE glasses.name = 'Highball'),
+	true
+);
```
Comment (BjoernPetersen) [inline sql/V2.4__Insert_drinks.sql]: The mock data should contain at least one non-alcoholic drink.

Comment (Chabare) [reply sql/V2.4__Insert_drinks.sql]: See here: 0c5fcdd42b26abdfaf65462f5d427a68829c587a

File: .gitignore (hunk 2 above)
Comment (Chabare) [reply .gitignore]: 711f5fa38e2965440fe57022917f710650f8778c

File: .circleci/config.yml (hunk 1 above)
Comment (Chabare) [reply .circleci/config.yml]: Added option in flyway.conf 711f5fa38e2965440fe57022917f710650f8778c

**Comment by Chabare** [comment]
Done :ok_hand: I don't see a reason either.

**Pull request merged**